docs/api-reference/next/image.md

```diff
@@ -384,6 +384,50 @@ export default function myImageLoader({ src, width, quality }) {
 
 Alternatively, you can use the [`loader` prop](#loader) to configure each instance of `next/image`.
 
+#### Akamai Loader
```

Comment:
Do we want to prefix this with saying these are the previous loaders from `next/legacy/image` so we aren't encouraging documenting every loader here?

Reply:
I was actually thinking we should document every loader here.

Is there a reason not to accept PR's for new loader docs here? Its not versioned so it seems safe.

Reply:
I moved these into a new document [c07770e](https://github.com/vercel/next.js/pull/46647/commits/c07770e2ebbe587efda7b78510eea883aaaa8dc0)